packages/material-ui-codemod/README.md

```diff
@@ -6,18 +6,17 @@
 [![npm downloads](https://img.shields.io/npm/dm/@material-ui/codemod.svg?style=flat-square)](https://www.npmjs.com/package/@material-ui/codemod)
 
 This repository contains a collection of codemod scripts based for use with
-[JSCodeshift](https://github.com/facebook/jscodeshift) that help update Material-UI
-APIs.
+[jscodeshift](https://github.com/facebook/jscodeshift) that help update Material-UI APIs.
 
-## Setup & Run
+## Setup & run
 
-- `npm install -D @material-ui/codemod@next` <!-- #default-branch-switch -->
-- `npx jscodeshift -t <path-to-codemod-script> <path>`
-  - Applies the transform script specified in `<path-to-codemod-script>` recursively to `<path>`.
+- Make the codemod locally available `npm install -D @material-ui/codemod@next` <!-- #default-branch-switch -->
```

Comment:
Stick with `npx` to avoid global installation

Reply:
npx doesn't work AFAIK. see point 4.

Reply:
> npx doesn't work AFAIK. see point 4.

It works perfectly fine if you follow the instructions. This is a documentation issue.